README.md

```
@@ -106,6 +106,63 @@ zora-agent audit --last 50    # last 50 actions
+**Irreversibility Scoring.** Every tool call is scored 0–100 before it executes. Writing a file: 20. A `git push` to origin: 70. Sending a Signal message: 80. Deleting a file: 95. Scores are configurable in `policy.toml`:
+
+```toml
+[actions.thresholds]
+warn      = 40   # log warning, allow
+flag      = 65   # pause and ask for approval
+auto_deny = 95   # block without asking
+```
+
+**Human-in-the-loop Approval.** When an action scores above the `flag` threshold, Zora pauses and sends you a Telegram message:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)